src/server/requirements.txt

```diff
@@ -3,8 +3,11 @@ flask-cors~=3.0.10
 flask-swagger-ui~=4.11.1
 gunicorn~=20.1.0
 jsonschema~=4.17.3
+numpy~=1.24.2
```

Comment:
Please create a new `requirements.txt` file in `model/` directory and move packages which are not related to the server.

Reply:
Done